app/components/to-review-project-card.js

```diff
@@ -20,11 +21,28 @@ export default class ToReviewProjectCardComponent extends Component {
     return moment(this.project.toReviewMilestoneActualEndDate).diff(moment(this.project.toReviewMilestoneActualStartDate), 'days');
   }
 
-  // hearingsSubmitted is a helper that iterates through each disposition
-  // and checks whether disposition has publichearinglocation and dateofpublichearing
-  @computed('project')
-  get hearingsSubmittedForProject() {
-    const dispositions = this.get('project.dispositions');
-    return hearingsSubmitted(dispositions);
+  // if the first object dcpPublichearinglocation and dcpDateofpublichearing properties are filled,
+  // then hearings have been submitted for that project
+  @computed('model.dispositions.firstObject.{dcpPublichearinglocation,dcpDateofpublichearing}')
+  get hearingsSubmitted() {
+    const location = this.get('model.dispositions.firstObject.dcpPublichearinglocation');
+    const date = this.get('model.dispositions.firstObject.dcpDateofpublichearing');
+    const locationAndDate = !!location && !!date;
+    return locationAndDate;
+  }
+
+  // if the first object dcpPublichearinglocation is "waived",
+  // then hearings have been waived
+  @computed('project.dispositions.firstObject.dcpPublichearinglocation')
```

Comment:
If you ever need to have this computed on a specific key of _each_ object in an array, you can use this: 

`project.dispositions.@each.dcpPublichearinglocation`

Sounds like it's not necessary here though :)